res/css/photon/confirm-dialog.css

```diff
@@ -45,3 +45,7 @@
   display: flex;
   gap: 8px;
 }
+
+.confirmDialogButtons > .photon-button {
+  flex: auto;
+}
```

Comment:
Without this change, they would shrink but not grow, which wasn't nice in the metainfo panel that has a fixed width. This has no effect in the other confirmation panel whose width depends on these buttons.